recipes/data-management/quantumleap/docker-compose.yml

```diff
@@ -70,11 +70,11 @@ services:
     #   - 4300:4300
     command: ["crate",
         "-Clicense.enterprise=false",
-        "-Cgateway.expected_nodes=3",
-        "-Cgateway.recover_after_nodes=2",
+        "-Cgateway.expected_nodes=${EXPECTED_NODES:-4}",
+        "-Cgateway.recover_after_nodes=${EXPECTED_NODES:-3}",
```

Comment:
I am not sure this should be the same as expected nodes. I suppose this should be the minimal number of nodes.

Reply:
good catch, fixed!